edx_proctoring/templates/proctored_exam/onboarding_error.html

```diff
@@ -0,0 +1,29 @@
+{% load i18n %}
+<div class="failure sequence proctored-exam {{onboarding_status}}" data-exam-id="{{exam_id}}">
+  <h3>
+    {% blocktrans %}
+      You must complete an onboarding exam before taking this proctored exam
+    {% endblocktrans %}
+  </h3>
+  <p>
+{% if onboarding_status == 'onboarding_missing' or onboarding_status ==  'onboarding_expired' %}
+    {% blocktrans %}
+    Please complete the <a href="{{onboarding_link}}">Onboarding Exam</a> before attempting this exam.
+    {% endblocktrans %}
+{% elif onboarding_status == 'onboarding_pending' %}
+    {% blocktrans %}
+    Your onboarding exam is being reviewed. Please allow 1-2 days before attempting this exam.
+    {% endblocktrans %}
+{% elif onboarding_status == 'onboarding_failed' %}
+    {% blocktrans %}
+      Your onboarding exam failed to pass all requirements. You can retake the <a href="{{onboarding_link}}">Onboarding Exam</a>.
+    {% endblocktrans %}
+{% endif %}
+  </p>
+  <p>
+    {% blocktrans %}
+      If you have questions about the status of your onboarding exam, contact {{ platform_name }} Support.
```

Comment:
As a potential later TODO once the details are ironed out, I think Olga suggested having a link to our support team here.